contracts/payloads/common/constants.sol

```diff
@@ -137,6 +142,11 @@ contract PayloadIGPConstants {
     address internal constant wstUSR_ADDRESS =
         0x1202F5C7b4B9E47a1A484E8B270be34dbbC75055;
 
+    address internal constant XAUT_ADDRESS = 
+        0x45804880De22913dAFE09f4980848ECE6EcbAf78;
```

Comment:
wrong way around, 0x45804880De22913dAFE09f4980848ECE6EcbAf78 is PAXG 

XAUT is https://etherscan.io/address/0x68749665FF8D2d112Fa859AA293F07A622782F38